var/spack/repos/builtin/packages/esmf/package.py

```diff
@@ -494,3 +495,6 @@ def install(self, pkg, spec, prefix):
             # build the python library
             python_builder = PythonPipBuilder(pkg)
             python_builder.install(pkg, spec, prefix)
+
+        if spec.satisfies("@8.9:"):
+            install_tree("cmake", self.prefix.cmake)
```

Comment:
Just curious what this line does. ESMF's `make install` target installs the `cmake ` dir into the installation prefix directory, but what does this line do?

Reply:
Oh. In this case, it's redundant. Let me try without.

Reply:
See https://github.com/JCSDA/spack/pull/532/commits/ea2ca37792646912193c498e858b22a29989f7ff